examples/step-19/doc/intro.dox

```diff
@@ -0,0 +1,449 @@
+@f}
+Because of the awkward dependence of the electric potential on the
+particle locations, we don't want to solve this as a coupled system
+but instead use a decoupled approach where we first solve for the
+potential in each time step and then the particle locations. (One
+could also do it the other way around, of course.) This is very
+much in the same spirit as we do in step-21, step-31, and step-32,
+to name just a few, and can all be understood in the context of
+the operator splitting methods discussed in step-58.
+
+So, if we denote by an upper index $n$ the time step, and if we
+use a simple time discretization for the ODE, then this means
+that we have to solve the following set of equations in each time
+step:
+@f{align*}{
+  -\epsilon_0 \Delta V^(n) &= \sum_i e\delta(\mathbf x-\mathbf x_i^{(n-1)})
+  \\
+  \frac{{\mathbf v}_i^{(n)}-{\mathbf v}_i^{(n-1)}}{\Delta t} &= \frac{e\nabla V^{(n)}}{m}
+  \\
+  \frac{{\mathbf x}_i^{(n)}-{\mathbf x}_i^{(n-1)}}{\Delta t} &= {\mathbf v}_i^{(n)}.
+@f}
+There are of course many better ways to do a time discretization (for
+example the simple [leapfrog scheme](https://en.wikipedia.org/wiki/Leapfrog_integration))
+but this isn't the point of the tutorial program, and so we will be content
+with what we have here. (We will comment on a piece of this puzzle in the
+<a href="#extensions">possibilities for extensions</a> section of this program,
+however.)
+
+There remains the question of how we should choose the time step size $\Delta t$.
+The limitation here is that the Particles::ParticleHandler class needs to
+keep track of which cell each particle is in. This is particularly an issue if
+we are running computations in parallel (say, in step-70) because in that case
+every process only stores those cells it owns plus one layer of "ghost cells".
+That's not relevant here, but in general we should make sure that over the
+course of each time step, a particle moves only from one cell to any
+of its immediate neighbors (face, edge, or vertex neighbors). If we can ensure
+that, then Particles::ParticleHandler is guaranteed to be able to figure out
+which cell a particle ends up in. To do this, a useful rule of thumb
+is that we should choose the time step so that for all particles the expected
+distance the particle moves by is less than one cell diameter:
+@f[
+  \Delta t \le \frac{h_i}{\|\mathbf v_i\|} \qquad\qquad \forall i,
+@f]
+or equivalently
+@f[
+  \Delta t \le \min_i \frac{h_i}{\|\mathbf v_i\|}.
+@f]
+Here, $h_i$ is the length of the shortest edge of the cell on which particle
+$i$ is located -- in essence, a measure of the size of a cell.
+
+On the other hand, a particle might already be at the boundary of one cell
+and the neighboring cell might be once further refined. So then the time to
+cross that *neighboring* cell would actually be half the amount above,
+suggesting
+@f[
+  \Delta t \le \min_i \frac{\tfrac 12 h_i}{\|\mathbf v_i\|}.
+@f]
```

Comment:
I would mention CFL here (because later you state that we already have discussed CFL).

Reply:
It's actually not a CFL condition, just a time step restriction. I opted to simply change the place where I mentioned "CFL" before, rather than doing it here.